utils.py

```diff
@@ -2111,8 +2111,18 @@ def minimal_realisation(a, b, c):
 
     return Aco, Bco, Cco
 
-
 def num_denom(A, symbolic_expr=False):
+    """
+    Produces a tuple containing the numerator and denominator polynomial
```

Comment:
All functions in Python which return multiple values return a tuple. It's usually not stated that way. We usually say "Returns the numerator and denominator polynomial"

Reply:
Addressed.